packages/cli/src/util/token.test.ts

```diff
@@ -35,19 +35,25 @@ describe("loadToken", () => {
       validToken,
     );
     const token = await loadToken(undefined, "valid-token.txt");
-
     expect(token).toBe(validToken);
   });
 
-  it("should load a valid token from an environment variable", async () => {
+  it("should load a valid token from the FOUNDRY_TOKEN environment variable", async () => {
     vi.stubEnv("FOUNDRY_TOKEN", validToken);
+    vi.stubEnv("FOUNDRY_SDK_AUTH_TOKEN", "");
+    expect(await loadToken()).toBe(validToken);
+  });
 
-    const token = await loadToken();
-    expect(token).toBe(validToken);
+  it("should load a valid token from the deprecated FOUNDRY_SDK_AUTH_TOKEN environment variable", async () => {
+    vi.stubEnv("FOUNDRY_TOKEN", "");
+    vi.stubEnv("FOUNDRY_SDK_AUTH_TOKEN", validToken);
+    expect(await loadToken()).toBe(validToken);
   });
 
   it("should throw an error if no token is found", async () => {
-    expect(() => loadToken()).rejects.toThrow("No token found.");
+    vi.stubEnv("FOUNDRY_TOKEN", "");
+    vi.stubEnv("FOUNDRY_SDK_AUTH_TOKEN", "");
```

Comment:
Isn't that equivalent to them not being set? And is it necessary set them to empty above as well?

Reply:
This is for running the tests locally where you may already have your own env set with these variables so the test can be affected by your local env. We make sure to stub out all the input env variables for the test to get a predictable result